utils/mock-data-loader/centralApi.js

```diff
@@ -1,7 +1,7 @@
 const axios = require('axios');
 require('dotenv').config();
 
-const urlRoot = process.env.DTFS_CENTRAL_API;
+const urlRoot = process.env.DTFS_CENTRAL_API_URL;
```

Comment:
same as above, this line can be combined with the one below

Reply:
Have re-factored.